src/lib/storage/value_segment.cpp

```diff
@@ -14,18 +14,20 @@ namespace opossum {
 
 template <typename T>
 AllTypeVariant ValueSegment<T>::operator[](const ChunkOffset chunk_offset) const {
-  throw std::runtime_error("Implement ValueSegment::operator[]");
+  if (chunk_offset >= this->size()) {
+    throw std::runtime_error("Vector out of bounds");
+  }
+  return valuesVector[chunk_offset];
```

Comment:
For variables, please use lowercase separated by underscores, e.g., `chunk_size`. (see `CONTRIBUTING.md`)